src/components/Tabs/Tabs.tsx

```diff
@@ -229,9 +229,8 @@ export const Tabs = ({
                                 id={`${tab.id}-btn`}
                                 className={merge([
                                     'tw-group tw-relative tw-mx-0 tw-py-4 tw-px-2 tw-w-max tw-cursor-pointer tw-flex tw-items-center tw-justify-center tw-whitespace-nowrap',
-                                    tab.disabled && 'tw-text-text-disabled',
-                                    !tab.disabled && 'hover:tw-text-text',
-                                    tab.id === activeItemId ? 'tw-font-medium tw-text-text' : 'tw-text-text-weak',
+                                    tab.id === activeItemId && 'tw-font-medium tw-text-text',
```

Comment:
@mnoleto  tab.id line should go after. Because the tab.disabled ternary overwrites the `tw-text-text` class and change it to `tw-text-text-weak`.

Reply:
Can we have a tab disabled and active at the same time?

Reply:
No you can't. but on line 233 it checks if disabled and if not disabled, it uses text-weak and it overwrites the active state with text-text styling.

Reply:
Done!